Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -475,13 +471,13 @@ private void UpdateInvalidLoginAttemptCount(User user, int newValue)
         private string GetLocalPasswordHash(User user)
         {
             return string.IsNullOrEmpty(user.EasyPassword)
-                ? _defaultAuthenticationProvider.GetEmptyHashedString(user)
+                ? null
                 : user.EasyPassword;
         }
 
         private bool IsPasswordEmpty(User user, string passwordHash)
         {
-            return string.Equals(passwordHash, _defaultAuthenticationProvider.GetEmptyHashedString(user), StringComparison.OrdinalIgnoreCase);
+            return string.IsNullOrEmpty(passwordHash);
```

Comment:
Why keep this method?

Reply:
because I didn't really notice, I've replaced it in the one space it was used.